src/com/sphoorthi/LibraryManagement/Book.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package LibraryManagement;
+
+import java.util.*;
+
+/**
+ *
+ * @author Sphoorthi Gaddam
+ */
+public class Book {
+    Set<String> books = new HashSet<String>();
```

Comment:
since book is the domain object it should have it property called as name which could be string.